libs/live-network/src/network.ts

```diff
@@ -198,9 +202,8 @@ export const newImplementation = async <T = unknown, U = unknown>(
       maxRetry: getEnv("GET_CALLS_RETRY"),
       retryCondition: error => {
         if (error && error.status) {
-          // A 422 shouldn't be retried without change as explained in this documentation
-          // https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
-          return error.status !== 422;
+          // not all status codes are retryable
+          return retryableHttpStatusCodes.includes(error.status);
```

Comment:
Be careful, everyone has not migrate to this call.
You might need to duplicate the code with `implementation` function.

Reply:
in my context i only need it fixed for `live-wallet`'s wallet sync code so at least it fixes this.
the benefits being that it limits the impact of the PR changes. only the new implementation users will benefit the evolution and i didn't break the behavior of the deprecated way.
when all users of live-network migrate to newImplementation they will then have the opportunity to verify this change is working 🤔